client-app/shared/notification/composables/useNotifications.ts

```diff
@@ -0,0 +1,78 @@
+import { readonly, shallowRef, triggerRef } from "vue";
+import _ from "lodash";
+import { INotification, INotificationExtended } from "@/shared/notification";
+
+const stack = shallowRef<INotificationExtended[]>([]);
+
+async function show(options: INotification) {
```

Comment:
If we have "show" method then we assume to have a "hide" method in opposite.
However we have "close" method, so we assume to have "open" method in opposite.

Maybe we should change naming to be more convenient?